build-system/server/app-index/index.js

```diff
@@ -29,6 +28,8 @@ const {getServeMode} = require('../app-utils');
 const {join} = require('path');
 const {renderTemplate} = require('./template');
 
+const fs = promisifyAll(require('fs'));
```

Comment:
Would the promise based fs apis be a better fit here?

https://nodejs.org/api/fs.html#fs_fs_promises_api

Reply:
Good catch. It certainly is!

I've removed `util-promisifyAll` in favor of the native promise APIs of `fs` and `fs-extra`.